examples/README.md

```diff
@@ -0,0 +1,89 @@
+# HPO with raytune POC on OpenShift AI
+
+This example demonstrates how to use raytune for hyperparameter optimisation using raytune on OpenShift AI.
```

Comment:
Would be good to explain the HPO example a bit more. What is the objective of this HPO exercise, what is the base example, what parameters are being tuned and what is the expected outcome that we are aiming for.